packages/manager/apps/hycu/src/components/FileUpload/FileUpload.component.tsx

```diff
@@ -31,14 +31,24 @@ export const FileInputField = ({
     if (file.size > MAX_FILE_SIZE) {
       setFileName('');
       control.setError(name, {
-        message: t('hycu_dashboard_upload_valid_file'),
+        message: t('hycu_dashboard_upload_file_too_big'),
+      });
+      return;
+    }
+    if (!file.name.endsWith('.req')) {
```

Comment:
To set in a constant to use also in translation key
```suggestion
    if (!file.name.endsWith(LICENSE_FILE_EXT)) {
```